recipe/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "arcticdb" %}
-{% set version = "1.6.1-rc1" %}
+{% set version = "1.6.1rc2" %}
 
 package:
   name: {{ name|lower }}
-  version: 1.6.1rc1
+  version: 1.6.1rc2
 
 source:
   url: https://github.com/man-group/ArcticDB/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 36dfe63ef0b0a4215b7ccdd2b322cdaf442ee3c617922d29c1a09db4a94fd2fd
+  sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
```

[The hash is apparently incorrect](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=758042&view=logs&j=d0d954b5-f111-5dc4-4d76-03b6c9d0cf7e&t=6d4b912b-175d-51da-0fd9-4d30fe1eb4e7&l=218), probably because the tag changed in between. Let's update it.
```suggestion
  sha256: f25ee584dfbe4b4cd5345c011bab8efab1836f2dc35019b12a9a9d2730e74bf7
```